server/recceiver/cfstore.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+
+import itertools
+
+from zope.interface import implements
+from twisted.internet import defer
+from twisted.application import service
+from twisted.enterprise import adbapi as db
+from channelfinder import ChannelFinderClient
+import interfaces
+import datetime
+
+# ITRANSACTION FORMAT:
+#
+# src = source address
+# addrec = records ein added ( recname, rectype, {key:val})
+# delrec = a set() of records which are being removed
+# infos = dictionary of client infos
+# recinfos = additional infos being added to existing records 
+# "recid: {key:vale}"
+#
+
+
+__all__  = ['CFProcessor']
+
+class CFProcessor(service.Service):
+    implements(interfaces.IProcessor)
+
+    def __init__(self, name, conf):
+        print "CF_INIT"
+        self.name,self.conf = name,conf
+        print "CONF"+str(conf)
+        
+        
+    def startService(self):
+        service.Service.startService(self)
+        self.running = 1
+        print "CF_START"
+        try:
+            '''
+            Using the default python cf-client.
+            The usr, username, and password are provided by the channelfinder._conf module.
+            '''
+            self.client = ChannelFinderClient()
+        except:
+            print 'Failed to create cf client'
```

Comment:
Swallowing unrecoverable exceptions like this is not good.  Should be re-raised.